lib/config/systemd.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"io"
+	"os"
+	"text/template"
+
+	"github.com/gravitational/trace"
+)
+
+const (
+	// DefaultEnvironmentFile is the default path to the env file
+	DefaultEnvironmentFile = "/etc/default/teleport"
```

Comment:
External packages that refer to this variable will refer to it as `config.DefaultEnvironmentFile`.

Nothing about this name makes it clear that it is specific to systemd configuration.

Should we rename it to something like `DefaultSystemdEnvironmentFile` instead?

(same goes for the other consts)